index.html

```diff
@@ -1740,14 +1738,15 @@ <h3><code>subprotocol</code> Vocabulary</h3>
           The <code>subprotocol</code> field is defined in [[!WOT-THING-DESCRIPTION]].
         </p>
         <p>
-          Currently, the supported values are <code>longpoll</code>, <code>websub</code> and <code>sse</code> defined
-          for HTTP.
+          Currently, the supported values are <code>longpoll</code>, <code>websub</code> and <code>sse</code> defined for HTTP.
           Subprotocols can be used for asynchronous event delivery or observing Properties.
         </p>
         <p>
           For WebSockets, the IANA-registered Websocket Subprotocols [[iana-web-socket-registry]] may be used.
         </p>
-
+        <p>
+          For CoAP, <code>"subprotocol":"cov:observe"</code> can be used to describe asynchronous observation operations as defined by [[RFC6741]]
```

Comment:
Just curious: I assume that `cov:observe` gets expanded to `http://www.example.org/coap-binding#observe`. What do `longpoll`, `websub`, and `sse` get expanded to?

Reply:
At the moment, they are defined as simple strings, see https://www.w3.org/2019/wot/hypermedia#forsubprotocol. Ideally, the HTTP vocabulary in RDF should define URIs for these sub-protocols.

Reply:
One issue with JSON-LD expansion (and transformation to RDF) is that we cannot have both strings and URIs in expanded form for a single context. Currently, in TDs, `cov:observes` will expand to the string literal `"cov:observe"` and not the expected URI.

Reply:
I see. I think implementers might get confused why all the other `cov:`'s get expanded but this one doesn't. Maybe `coap-observe` or something similar would be preferable to `cov:observe`?

Reply:
In the case `coap-observe`, this should be then defined in the TD vocabulary?